examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/attestation/ExampleAttestationTrustStoreDelegate.kt

```diff
@@ -0,0 +1,48 @@
+package com.google.chip.chiptool.attestation
+
+import android.util.Base64
+import chip.devicecontroller.AttestationTrustStoreDelegate
+import chip.devicecontroller.ChipDeviceController
+import java.util.*
+
+class ExampleAttestationTrustStoreDelegate(val chipDeviceController: ChipDeviceController) :
+  AttestationTrustStoreDelegate {
+
+  private val paaCerts = arrayListOf(TEST_PAA_FFF1_Cert, TEST_PAA_NOVID_CERT)
+
+  override fun getProductAttestationAuthorityCert(skid: ByteArray): ByteArray? {
+    for (paaCert in paaCerts) {
```

Comment:
Suggestion: Use mapping functions here:

```
return paaCerts
  .map { Base64.decode(it, Base64.DEFAULT) }
  .map(chipDeviceController::extractSkidFromPaaCert)
  .firstOrNull { it == skid }
```

Reply:
Still try to understand this syntax, it looks to me this return skidFromPaaCert instead of paaCertBytes, no?

Reply:
You're right, I missed that, instead it would be:

```
return paaCerts
  .map { Base64.decode(it, Base64.DEFAULT) }
  .firstOrNull { cert -> chipDeviceController.extractSkidFromPaaCert(cert) == skid }
```